content/config/daemon/troubleshoot.md

```diff
@@ -368,6 +368,171 @@ To turn off `dnsmasq` on RHEL, CentOS, or Fedora:
 
+### Docker networks disappearing
+
+If a Docker network, such as the `docker0` bridge or a custom network, randomly
+disappears or otherwise appears to be working incorrectly, it could be because
+another service is interfering with or modifying Docker interfaces. Tools that
+manage networking interfaces on the host are known to sometimes also
+inappropriately modify Docker interfaces.
+
+Refer to the following sections for instructions on how to configure your
+network manager to set Docker interfaces as un-managed, depending on the
+network management tools that exist on the host:
+
+- If `netscript` is installed, consider [uninstalling it](#uninstall-netscript)
+- Configure the network manager to [treat Docker interfaces as un-managed](#un-manage-docker-interfaces)
+- If you're using Netplan, you may need to [apply a custom Netplan configuration](#prevent-netplan-from-overriding-network-configuration)
+
+#### Uninstall `netscript`
+
+If `netscript` is installed on your system, you can likely fix this issue by
+uninstalling it. For example, on a Debian-based system:
+
+```console
+$ sudo apt-get remove netscript-2.4
+```
+
+#### Un-manage Docker interfaces
+
+In some cases, the network manager will attempt to manage Docker interfaces by
+default. You can try to explicitly flag Docker networks as un-managed by
+editing your system's network configuration settings.
+
+{{< tabs >}}
+{{< tab name="NetworkManager" >}}
+
+If you're using `NetworkManager`, edit your system network configuration under
+`/etc/network/interfaces`
+
+1. Create a file at `/etc/network/interfaces.d/20-docker0` with the following
+   contents:
+
+   ```text
+   iface docker0 inet manual
+   ```
+
+   Note that this example configuration only "un-manages" the default `docker0`
+   bridge, not custom networks.
+
+2. Restart `NetworkManager` for the configuration change to take effect.
+
+   ```console
+   $ systemctl restart NetworkManager
+   ```
+
+3. Verify that the `docker0` interface has the `unmanaged` state.
+
+   ```console
+   $ nmcli device
+   ```
+
+{{< /tab >}}
+{{< tab name="systemd-networkd" >}}
+
+If you're running Docker on a system using `systemd-networkd` as a networking
+daemon, configure the Docker interfaces as un-managed by creating configuration
+files under `/etc/systemd/network`:
+
+1. Create `/etc/systemd/network/docker0.network` with the following contents:
+
+   ```ini
+   # Ensure that the 'docker0' interface is un-managed
+   
+   [Match]  
+   Name=docker0
+   
+   [Link]
+   Unmanaged=yes
+   ```
+
+2. Create `/etc/systemd/network/dockerbridge.network` with the following contents:
+
+   ```ini
+   # Ensure that the docker interfaces 'br-' are un-managed
+   
+   [Match]  
+   Name=br-*
+   
+   [Link]
+   Unmanaged=yes
+   ```
+
+3. Create `/etc/systemd/network/veth.network` with the following contents:
+
+   ```ini
+   # Ensure that veth interfaces are un-managed
+   
+   [Match]  
+   Driver=veth*
+   
+   [Link]
+   Unmanaged=yes
+   ```
```

Comment:
https://www.freedesktop.org/software/systemd/man/latest/systemd.network.html#Name=

> A whitespace-separated list of shell-style globs matching the device name ...

```suggestion
1. Create `/etc/systemd/network/docker.network` with the following contents:

   ```ini
   # Ensure that the Docker interfaces are un-managed
   
   [Match]  
   Name=docker0 br-* veth*
   
   [Link]
   Unmanaged=yes
   ```
```

Reply:
thank you 😅